### PR TITLE
Adding the initial notes for the Governance planning

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

Partially addresses #2035 

* Add the sections and bullet points for the Governance plan

### List of changes proposed in this PR (pull-request)

* Adding a draft file for the governance
* Will archive some outdated files from the project management folder

### What should a reviewer concentrate their feedback on?

- [ ] What is missing from the current structure
- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

File: project_management/governance-docs/2021-07-governance-structure.md [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+- See: https://hackmd.io/@turingway/demo-intro
+- *Policies and procedures*
+    - CoC
+    - Acknowledgements
+    - Ways of working
+- *Reporting and communication*
+    - Monthly updates: newsletter
+    - Quarterly: TPS report
+    - Yearly: review and impact report
+    - Communication: email (may need update), Slack, Twitter
```
Comment (EKaroune) [inline project_management/governance-docs/2021-07-governance-structure.md:125]: add google calendar

File: project_management/governance-docs/2021-07-governance-structure.md [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+- *Infrastructure: technology and platforms*
+    - JupyterBook
+    - Netlify
+    - GitHub
+    - Binder (partner)
+    - ?
```
Comment (EKaroune) [inline project_management/governance-docs/2021-07-governance-structure.md:131]: Hackmd

Comment (malvikasharan) [reply project_management/governance-docs/2021-07-governance-structure.md:134]:
```suggestion
    - HackMD
```

File: project_management/governance-docs/2021-07-governance-structure.md (hunk 1 above)
Comment (malvikasharan) [reply project_management/governance-docs/2021-07-governance-structure.md:128]:
```suggestion
    - Communication: email (may need update), Slack, Twitter, Google calendar, Gitter (mostly to direct people to different channels where we are engaging in active discussions)
```

**Comment by sgibson91** [comment]
Just wanted to bring in my point from Slack, though not entirely sure where it fits in to this document specifically.

Leaders of working groups or any group working on a specific effort need a _lightweight_ process by which to surface proposals/requests for feedback to the steering committee/appropriate governance body to assess whether the impact of a particular project-wide change is within the Turing Way's values and in the best interests of the projects.

A couple of discussions I've been involved in where knowing the stance of the core team would have been helpful include:

- Should the Turing Way website have it's own domain name?
- How do we handle the growing size of the repo? Which technical solution solves the problem without being too technical that we isolate a vast majority of the community?

Project Jupyter has [Jupyter Enhancement Proposals](https://jupyter.org/enhancement-proposals/29-jep-process/jep-process.html as their process - though I do not think this is in anyway "lightweight", so not the greatest example.

**Comment by aleesteele** [comment]
Planning to work on some of these governance issues during the May book dash, refrencing additional work being done at OLS: https://github.com/open-life-science/ols-5/issues/31

**Comment by bsipocz** [comment]
We are in the middle of reorganizing the repository and as part of that work have moved some of the documents into a new repo. This has introcude code conflicts above, and therefore I am closing this pull request.

Could you please reopen this PR in the new PM repo: https://github.com/the-turing-way/project-management

**Comment by JimMadge** [comment]
This PR has been moved [to here](https://github.com/the-turing-way/project-management/pull/3) :pray:.

**Pull request closed without merging**